packages/modules/order/src/services/actions/receive-return.ts

```diff
@@ -1,39 +1,44 @@
 import { Context, OrderTypes } from "@medusajs/types"
-import { MathBN, ReturnStatus } from "@medusajs/utils"
+import { MathBN, ReturnStatus, promiseAll } from "@medusajs/utils"
 import { OrderChangeType } from "@types"
 import { ChangeActionType } from "../../utils"
 
-export async function receiveReturn(
-  this: any,
-  data: OrderTypes.ReceiveOrderReturnDTO,
-  sharedContext?: Context
-) {
-  const returnEntry = await this.retrieveReturn(
-    data.return_id,
+async function retrieveReturn(service, returnId, sharedContext) {
```

Comment:
thought: I guess we can just use the service method directly in the module service, now that we no longer decorate with return items. No need for this util

Reply:
Yeah sure, I just split everything and composed the main methods like workflows definition haha...